### PR TITLE
Updates to currentBidInfo components

### PR DESCRIPTION
**Pull request by oxaudo**
Adding some specs as well.

Not sure how to get data into `myLotStanding` locally. But just came up with some values in test data and looks like it's displaying properly.

### CODE REVIEW

**Comment by oxaudo** [comment]
And here are the test cases I've added to storybook:
screen shot 2018-12-12 at 3 02 19 pm

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit b35eeea3f469812679dd3da4eb7b246cf051d893

https://deploy-preview-1686--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.tsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { Box, Flex, Sans, Serif } from "@artsy/palette"
+import { LoosingBid } from "Assets/Icons/LoosingBid"
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.tsx]: typo `Losing` 🐞

Comment (oxaudo) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.tsx]: Oh English! https://www.google.com/search?q=How+do+you+spell+loosing+or+losing%3F&sa=X&ved=2ahUKEwiptIXmjJvfAhUSmuAKHeGxAAQQzmd6BAgGEAk&biw=1166&bih=921

File: src/Apps/Artwork/Components/__tests__/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.test.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
+import {
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/__tests__/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.test.tsx]: Awesome!

**Pull request merged**